code/modules/antagonists/clock_cult/structure/eminence_beacon.dm

```diff
@@ -21,7 +21,7 @@
 		to_chat(user, "<span class='brass'>The Eminence has already been released.</span>")
 		return
 	var/option = alert(user,"Who shall control the Eminence?",,"Yourself","A ghost", "Cancel")
-	if(option == "Cancel")
+	if(option != "A ghost")
 		return
 	else if(option == "Yourself")
```

Comment:
Doesn't `!= "A ghost"` make it impossible for the else if to run? Because it will just call return before then.
```suggestion
	if(option != "A ghost" && option != "Yourself")
		return
	else if(option == "Yourself")
```
You can either do this or swap the contents of the if and else if checks.

Reply:
Did the first thing and added an //NSV13 comment to the modified one